objgraph.py

```diff
@@ -662,14 +679,18 @@ def _show_graph(objs, edge_func, swap_source_target,
         f = codecs.open(filename, 'w', encoding='utf-8')
         dot_filename = filename
     else:
-        fd, dot_filename = tempfile.mkstemp(prefix='objgraph-',
-                                            suffix='.dot', text=True)
-        f = os.fdopen(fd, "w")
-        if getattr(f, 'encoding', None):
-            # Python 3 will wrap the file in the user's preferred encoding
-            # Re-wrap it for utf-8
-            import io
-            f = io.TextIOWrapper(f.detach(), 'utf-8')
+        if IS_INTERACTIVE:
```

Comment:
Can you use `elif` here instead of nesting an `if` inside the `else`?  I think it'll also make the diff smaller (no indentation changes for the write-to-file branch).